source/wp-content/themes/wporg-parent-2021/patterns/two-columns-with-cta.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * Title: 2 Columns with CTA
+ * Slug: wporg-parent-2021/two-columns-with-cta
+ * Categories: columns, buttons
+ */
+
+?>
+<!-- wp:columns {"align":"full","style":{"border":{"top":{"color":"var:preset|color|light-grey"}}},"className":"is-style-default"} -->
+<div class="wp-block-columns alignfull is-style-default" style="border-top-color:var(--wp--preset--color--light-grey)"><!-- wp:column {"width":"100%","style":{"spacing":{"padding":{"top":"80px","right":"100px","bottom":"100px","left":"100px"}}}} -->
+<div class="wp-block-column" style="padding-top:80px;padding-right:100px;padding-bottom:100px;padding-left:100px;flex-basis:100%"><!-- wp:heading {"level":4} -->
+<h4>Download and install yourself</h4>
+<!-- /wp:heading -->
+
+<!-- wp:paragraph -->
+<p>For anyone comfortable getting their own hosting and domain.</p>
+<!-- /wp:paragraph -->
+
+<!-- wp:buttons -->
+<div class="wp-block-buttons"><!-- wp:button {"style":{"spacing":{"padding":{"right":"32px","left":"32px"}}}} -->
+<div class="wp-block-button"><a class="wp-block-button__link wp-element-button" style="padding-right:32px;padding-left:32px">Download WordPress 6.0</a></div>
```

Comment:
Default Button background color doesn't match designs. Not sure whether we would adjust button padding in theme settings too.

Reply:
I updated the button styles with #18, so the default styles should match now - let me know if it's not right, though 🙂